### PR TITLE
add trying out

### PR DESCRIPTION
**Pull request by pruwt**
@adrianmurage @Eze-kiel-a added trying out span

### CODE REVIEW

File: views/home.hbs [hunk 1]
```diff
@@ -26,6 +26,7 @@
             <table id="content">
                 <span>we're deleting</span>
                 <span> everything</span>
+                <span>trying out git and github</span>
```
Comment (adrianmurage) [inline views/home.hbs:29]: I really don't like your work
DO IT AGAIN. BUT BETTERRRRR

**Review by adrianmurage — CHANGES_REQUESTED** [review]
do it again!

File: views/home.hbs (hunk 1 above)
Comment (pruwt) [reply views/home.hbs:29]: no

**Comment by adrianmurage** [comment]
okay live your life

**Review by adrianmurage — APPROVED** [review]

**Pull request merged**